components/heatmap/HeatMap.vue

```diff
@@ -233,9 +233,8 @@ export default {
     },
     passing_plate_colors: function () {
       return this.well_values[this.display_option].data.map((well) => {
-        const well_data = well.slice(-MAX_NUM_DATAPOINTS_FOR_MEAN);
```

Comment:
Was this unnecessary because the slicing is already done in the store?

Reply:
@tannermpeterson yes